scheduler/batch.go

```diff
@@ -53,7 +53,7 @@ func (s *BatchSettings) getBatcher(ctx context.Context, res chan<- message.SyncM
 			res:     res,
 			maxRows: s.MaxRows,
 			timeout: s.Timeout,
-			logger:  logger.With().Int("max_rows", s.MaxRows).Dur("timeout", s.Timeout).Logger(),
```

Comment:
Maybe we should keep it as `Dur` to avoid the `.String()` allocation? I think the logger uses Milliseconds, so we could change the field name to `timeout_ms`